app/controllers/user_sessions_controller.rb

```diff
@@ -50,7 +50,13 @@ def deliver_access_details
   end
 
   def user_session
-    @user_session ||= UserSession.new(params[:user_session])
+    @user_session ||= UserSession.new(user_session_params.to_unsafe_hash)
+  end
+
+  def user_session_params
+    params.require(:user_session).permit(:reference, :password, :email_address)
```

Comment:
user_session is optional - if it is missing we simply want an empty set of params.  This is the original design, just had to stick with it